schema/GeneticVariant_Alt_ID_Database_properties.mcf

```diff
@@ -0,0 +1,1401 @@
+# Generated by script format_dbSNP_alt_ID_database_property_schema.py
+
+Node: dcid:fanconi_Anaemia_Mutation_Database_(FANCM)_ID
+name: "fanconi_Anaemia_Mutation_Database_(FANCM)_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database fanconi_Anaemia_Mutation_Database_(FANCM) for a genetic variant."
+
+Node: dcid:tIDEX_University_of_British_Columbia_ID
+name: "tIDEX_University_of_British_Columbia_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database tIDEX_University_of_British_Columbia for a genetic variant."
+
+Node: dcid:shworak_lab_George_Washington_University_ID
+name: "shworak_lab_George_Washington_University_ID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:GeneticVariant
+description: "The ID used by database shworak_lab_George_Washington_University for a genetic variant."
+
+Node: dcid:lDLR-LOVD_British_Heart_Foundation_ID
```

Comment:
ldlr_LOVD or lDLR_LOVD? To be consistent about the use of "_"

Reply:
changed to camel case